src/vue/navigation/Passport.vue

```diff
@@ -64,13 +64,17 @@ import { vuexTypes } from '@/vuex'
 import { mapGetters, mapMutations } from 'vuex'
 import { vueRoutes } from '@/vue-router/routes'
 import { handleClickOutside } from '@/js/helpers/handle-click-outside'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+import VerificationFormMixin from '@/vue/mixins/verification-form.mixin'
 
 export default {
   name: 'passport',
+  mixins: [VerificationFormMixin],
```

Comment:
you don’t need this mixin, all the used methods can be mapped the same way they mapped in the mixin